packages/ui/src/components/va-input-wrapper/VaInputWrapper.vue

```diff
@@ -236,41 +226,30 @@ export default defineComponent({
     display: flex;
     align-items: center;
     flex: 1;
-    min-height: var(--va-input-min-height);
+    min-height: var(--va-input-wrapper-min-height);
     width: var(--va-form-element-default-width);
     border-color: var(--va-input-wrapper-border-color);
     border-style: solid;
-    border-width: var(--va-input-border-width);
-    padding: 0 var(--va-input-content-horizontal-padding);
+    border-width: var(--va-input-wrapper-border-width);
+    border-radius: var(--va-input-wrapper-border-radius);
+    padding: 0 var(--va-input-wrapper-horizontal-padding);
+    gap: var(--va-input-wrapper-items-gap);
     z-index: 0;
     overflow: hidden;
+    color: v-bind(textColorComputed);
+
+    @include va-background(var(--va-input-wrapper-background), var(--va-input-wrapper-background-opacity), -1);
 
     input,
     textarea {
```

Comment:
is it OK that I can't see scroll for this?
![image](https://github.com/epicmaxco/vuestic-ui/assets/35342240/8bfe530e-063f-4719-8b24-ddd4b9b90591)